scientific_library/tvb/simulator/simulator.py

```diff
@@ -376,9 +408,9 @@ def __call__(self, simulation_length=None, random_state=None):
 
         self.calls += 1
         if simulation_length is not None:
-            self.simulation_length = float(simulation_length)
+            self.simulation_length = simulation_length
```

Comment:
should keep the conversion to float here

Reply:
I don't remember removing it. Probably I just copied-pasted an older version.